spec_7/dateConverter.js

```diff
@@ -0,0 +1,63 @@
+import { identifiers, months } from './resources'
+import fs from 'fs'
+
+const converter = {
+
+  getRawDates: filePath => {
+    const raw_dates = fs.readFileSync( filePath, 'utf8' )
+    return raw_dates.split('\n')
+  },
+
+  trimCommas: word => word.replace( /,/g, '' ),
```

Comment:
This regex would remove all commas from within a string as well - not sure if that was intentional or not.  (The function name `trimCommas` makes me think the intent is to remove a comma from the beginning or end of the string.)